build.sh

```diff
@@ -1,5 +1,30 @@
-#!/bin/bash -e
+#!/usr/bin/env bash
+
+# This script is a modification of the build for docke2aci
+# https://github.com/appc/docker2aci/blob/master/build.sh
+
+set -e
+
+ORG_PATH="github.com/kinvolk"
+REPO_PATH="${ORG_PATH}/tcd"
+
+if [ ! -h gopath/src/${REPO_PATH} ]; then
+    mkdir -p gopath/src/${ORG_PATH}
+    ln -s ../../../.. gopath/src/${REPO_PATH} || exit 255
+fi
+
+export GOBIN=${PWD}/bin
+export GOPATH=${PWD}/gopath:${PWD}/Godeps/_workspace
+
+eval $(go env)
+go get -d
```

Comment:
About the "go get -d": please add a comment to say this needs to be removed after this is vendored, to be sure not to forget.